packages/oauth/src/form/login/form-auth-login-service.ts

```diff
@@ -58,7 +66,12 @@ class FormAuthLoginService {
     const currentUri = new URL(window.location.href)
     const searchParams: URLSearchParams = currentUri.searchParams
     if (searchParams.has('redirect_uri')) {
-      return searchParams.get('redirect_uri') as string
+      const uri = new URL(searchParams.get('redirect_uri') as string)
+      if (validateRedirectURI(uri)) {
+        return sanitizeUri(uri)
+      } else {
+        log.error('invalid uri')
```

Comment:
Looks like you misunderstood my feedback comment. What I was saying is something like:
```
log.error('invalid uri:', uri)
```
Does it make sense?